Installer/OSX/launchers/duplicati-cli.m

```diff
@@ -0,0 +1,13 @@
+#import "run-with-mono.h"
+
+NSString * const ASSEMBLY = @"Duplicati.CommandLine.exe";
+NSString * const APP_NAME = @"Duplicati.CommandLine";
+int const MONO_VERSION_MAJOR = 4;
```

Comment:
I think the changes in pull request #3844 that updated the target framework to 4.6.2 require a minimum mono version of  5.0.0 (see comment [here](https://github.com/duplicati/duplicati/pull/3844#issuecomment-525982565)).  Does this value need to be consistent with that?

Reply:
Yes, I will update it to 5.0.